src/components/body/bodyparts/EduAndSkills.js

```diff
@@ -1,51 +0,0 @@
-import Card from "../../general/Card";
-import "./EduAndSkills.css";
-import Icon from "../../general/Icon";
-import ReactJSLogo from "../../../thumbnails/icons/ReactJSLogo.png";
-import CSS3Logo from "../../../thumbnails/icons/CSS3Logo.png";
-import HTML5Logo from "../../../thumbnails/icons/HTML5Logo.png";
-import JavaScriptLogo from "../../../thumbnails/icons/JavaScriptLogo.png";
-import ReduxLogo from "../../../thumbnails/icons/ReduxLogo.png";
-import TypeScriptLogo from "../../../thumbnails/icons/TypeScriptLogo.png";
-import RStudioLogo from "../../../thumbnails/icons/RStudioLogo.png";
-import SQLLogo from "../../../thumbnails/icons/SQLLogo.png";
-import JavaLogo from "../../../thumbnails/icons/javaLogo.png"
-import Content from "../../general/Content";
-
-const EduAndSkills = () => {
-  const contents = [{
-    // title: '',
-    // date:'',
-    // role: '',
-    description: ["• University of Melbourne, Bachelor of Commerce (Actuarial Sciences 2022 Graduate)"]
-  }]
-  const contentList = contents.map((x) => <Content object={x}/>)
-  return (
-    <div className="eduandskills">
-      <Card>
-        <h2 className="education">Education</h2>
-        {contentList}
-        <div className="skills-container">
-          <h2>Skills</h2>
-          <div className="skill-icons">
```

Comment:
If you are copy and pasting many identical blocks with different info, then it calls for a variable array, and looping through the elements